system/settings/settings.xml

```diff
@@ -1411,7 +1411,7 @@
         </setting>
         <setting id="pvrplayback.confirmchannelswitch" type="boolean" label="19281" help="36231">
           <level>1</level>
-          <default>false</default>
+          <default>true</default>
```

Comment:
> short: the setting we're discussing here has nothing to do with channel switching via numeric input anymore

it changes a well known behaviour. if not required for this PR, and if you really want to change that behaviour, it should be sent as a separate PR